book/routing.rst

```diff
@@ -656,8 +657,10 @@ requirements can easily be added for each parameter. For example:
         // ...
 
         /**
-         * @Route("/blog/{page}", defaults={"page": 1}, requirements={
-         *     "page": "\d+"
+         * @Route("/blog/{page}", name="blog",
+         *     defaults={"page": 1},
+         *     requirements={
+         *         "page": "\d+"
          * })
```

Comment:
If we're going to do this, why not move `name` onto the next line too? And now, the final `}` should be on the line above, not on the same line as the `)`